homeassistant/components/rova/config_flow.py

```diff
@@ -0,0 +1,93 @@
+"""Config flow for the Rova platform."""
+
+from typing import Any
+
+from requests.exceptions import ConnectTimeout, HTTPError
+from rova import rova
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import (
+    CONF_HOUSE_NUMBER,
+    CONF_HOUSE_NUMBER_SUFFIX,
+    CONF_ZIP_CODE,
+    DEFAULT_NAME,
+    DOMAIN,
+)
+
+
+class RovaConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle Rova config flow."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self._errors: dict[str, str] = {}
+
+    def _check_area(self, zip_code: str, number: str, suffix: str) -> bool:
+        """Check if rova collects garbage at this location."""
+        api = rova.Rova(zip_code, number, suffix)
+
+        try:
+            response = api.is_rova_area()
+            if not response:
+                self._errors = {"base": "invalid_rova_area"}
+                return False
+        except (ConnectTimeout, HTTPError):
+            self._errors = {"base": "could_not_connect"}
+            return False
+        return True
```

Comment:
I think you can just inline this into the flow instead of creating it a separate function. This will save you using the class variable